ui-components/autocompleter.md

```diff
@@ -45,6 +45,7 @@ The `fetch` function is called whenever the trigger `char` is pressed and the `m
 | Name | Value | Description |
 | ---- | ----- | ----------- |
 | hide | () => void | Hides the autocompleter menu. |
+| reload| (fetchOptions: Record<string, any>) => void | Hides the autocompleter menu and fetches new menu items. The additional `fetchOptions` will be passed to the `fetch` callback. |
```

Comment:
Re: my first comment, is fetchOptions optional here too, or needs to be defaulted? Also, I think "additional" isn't necessary?

Reply:
Removed `additional` as you're right it's not needed. Also it's not optional as per the types, as if a dev is reloading then you'd want to get different data so you'd want to pass different fetch options.